docs/custom-audio/integrating-wwise-to-unreal.md

```diff
@@ -0,0 +1,189 @@
+---
+sidebar_position: 2
+---
+
+# Integrating Wwise into Unreal Engine
+
+With a proper Wwise integration, you can add new sounds, do some editing like random pitch shifting without editing every sound file or even randomize the sounds played, make new tracks, tie the sound levels to the in game sliders, and make the sound work on both Steam/Epic platforms and Gamepass version.
+
+## The Integration Process
+
+:::caution Make sure you have time and drive space set for this
+This will take some time and about 10-12 gigabytes of free space. You'll be building the unreal engine and downloading 2 SDKs. Depending on your computer and net speed, it can take a bit.
+:::
+
+### Prerequisites
+
+- [Visual Studio 2019](https://visualstudio.microsoft.com/vs/older-downloads/)
```

Comment:
You don't need 2019, 2017-2022 should work.

Reply:
Fixed, plus linked UE docs for needed submodules.